SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -34,62 +34,66 @@ struct ViewListingView: View {
         
         ZStack {
             Color("BackgroundGrey").ignoresSafeArea()
-            VStack(alignment: .leading) {
-                
-                GeometryReader { geometry in
-                    ImageCarouselView(numberOfImages: self.numberOfImages) {
-                        ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
-                            Image(uiImage: image)
-                                .resizable()
-                                .scaledToFit()
-                                .frame(width: geometry.size.width, height: 250)
-                                .aspectRatio(contentMode: .fill)
+            ScrollView {
+                VStack(alignment: .leading) {
+                    
+                    GeometryReader { geometry in
+                        ImageCarouselView(numberOfImages: self.numberOfImages) {
+                            ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
+                                Image(uiImage: image)
+                                    .resizable()
+                                    .scaledToFit()
+                                    .frame(width: geometry.size.width, height: 250)
+                                    .aspectRatio(contentMode: .fill)
+                            }
                         }
                     }
```

Comment:
add `.frame(height:295)` at line 51. [See Here](https://www.hackingwithswift.com/quick-start/swiftui/how-to-provide-relative-sizes-using-geometryreader)